assets/src/containers/DashboardAppBar.js

```diff
@@ -16,7 +16,7 @@ const styles = theme => ({
     flexGrow: 1
   },
   button: {
-    margin: theme.spacing.unit
+    margin: theme.spacing()
```

Comment:
What's the default here? 1 (times the 8 multiplier)? If it's 1, it may be more transparent to add the 1 as input. I was looking at [this doc page](https://material-ui.com/customization/spacing/#spacing), it wasn't clear to me.

Reply:
> All the title header for the views have color contract issues

I wonder if we need to import some typography: https://material-ui.com/components/typography/

Reply:
I'm not sure, I put in `1` and it didn't make a visual difference - I've committed that now.